MobSF/utils.py

```diff
@@ -628,10 +628,10 @@ def get_proxy_ip(identifier):
 
 def is_safe_path(safe_root, check_path):
     """Detect Path Traversal."""
-    safe_root = os.path.normpath(safe_root)
-    check_path = os.path.normpath(check_path)
+    safe_root = os.path.realpath(os.path.normpath(safe_root))
+    check_path = os.path.realpath(os.path.normpath(check_path))
     return os.path.commonprefix(
-        (os.path.realpath(check_path), safe_root)) == safe_root
+        (os.path.realpath(check_path), safe_root))) == safe_root)
```

Comment:
Why `realpath()` again?
See: https://mobsf.github.io/docs/#/tests for linting and running tests locally